SUMMARY.md

```diff
@@ -186,9 +218,23 @@ Biscuit {
 }
 ```
 
+Let's unpack what's displayed here:
+
+ - `symbols` carries a list of symbols used in the biscuit.
+ - `authority` carries information provided by the token creator. It gives the initial scope of the bicuit.
+ - `blocks` carries a list of blocks, which can refine the scope of the macaroon.
+
+Here, `authority` provides the initial block, which can be refined in subsequent blocks.
+
 A block comes with new symbols it adds to the system (there's a default symbol
 table that already contains values like `#authority` or `#operation`). It can
-contain facts, rules and caveats.
+contain facts, rules and caveats. A block contains:
+
+ - `symbols` a block can introduce new symbols (TODO are those symbols usable in following blocks?)
+ - `context` TODO
+ - `facts` each block can define new facts (but only only `authority` can define facts mentioning `#authority`)
+ - `rules` each block can define new rules
```

Comment:
only the `authority` block can define rules that generate facts with `#authority`